vscode-powershell.build.ps1

```diff
@@ -47,22 +47,18 @@ task RestoreEditorServices -If (Get-EditorServicesPath) {
             Invoke-Build Build (Get-EditorServicesPath) -Configuration $Configuration
         }
         "Release" {
-            # When releasing, we ensure the bits are not symlinked but copied,
-            # and only if they don't already exist.
-            if ((Get-Item ./modules -ErrorAction SilentlyContinue).LinkType -eq "SymbolicLink") {
-                Write-Build DarkRed "Deleting PSES symbolic link"
+            # With VSCE --follow-symlinks support, we can now use symlinks in Release mode too.
```

Comment:
This no longer needs to be split between debug and release configurations since the logic will be shared.